extensions/kubectl-cli/src/download.spec.ts

```diff
@@ -66,6 +66,19 @@ const releases: KubectlGithubReleaseArtifactMetadata[] = resultREST.map(
   },
 );
 
+const unsortedResultREST = JSON.parse(
+  fsActual.readFileSync(path.resolve(__dirname, '../tests/resources/unsorted-release-all.json'), 'utf8'),
```

Comment:
You can probably import the json using _import * as unsortedResultREST from `../tests/resources/unsorted-release-all.json`_

Reply:
It is same as the "sorted" json in code above ^^